scripts/sandbox.ts

```diff
@@ -329,6 +358,8 @@ export async function sandbox(optionValues: OptionValues<typeof options>) {
     );
     mainConfig.setFieldValue(['core', 'disableTelemetry'], true);
 
+    if (builder === '@storybook/builder-vite') forceViteRebuilds(mainConfig);
```

Comment:
Why is this in a separate spot from where the `webpackFinal` is being added?

Reply:
The `webpackFinal` bit is added in order to allow us to link in stories, so it sort of feels correct to do the webpack change in the `addStories` function. But I can see the argument why that's confusing. WDYT?

Reply:
I don't feel strongly, but it would be kind of nice to have one spot to go to for configuring the two builders differently from each other.  Especially if we think there may be other changes in viteFinal or webpackFinal for other purposes than what they're doing right now.

Reply:
Done, I think it's an improvement thanks!